draft-irtf-cfrg-hpke.md

```diff
@@ -1867,6 +2004,16 @@ These test vectors are also available in JSON format at {{TestVectors}}.
 
 ## DHKEM(X25519, HKDF-SHA256), HKDF-SHA256, AES-128-GCM
 
+{::comment}
+[rfced] In Appendix A, there are a number of lines that exceed the
+69-character limit for a sourcecode element. Please review and let us
+know how these lines can be modified.
+
+Our understanding is that test vectors should be marked as <sourcecode>,
+with or without the type set to “test-vectors”.  If you strongly prefer
+<artwork>, please let us know.
```

Comment:
They exceed the 69-character limit because you modified them to do so!

I don't have an opinion on `<sourcecode>` vs. `artwork`; if you use the already-wrapped values provided, they should fit within the 69-character limit.